Exiled.API/Features/Toys/Primitive.cs

```diff
@@ -0,0 +1,153 @@
+// <copyright file="Primitive.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Toys
+{
+    using System;
+    using AdminToys;
+    using Mirror;
+    using UnityEngine;
+
+    /// <summary>
+    /// A wrapper class for primitive Admin Toys.
+    /// </summary>
+    public class Primitive
+    {
+        private bool collidable = true;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Primitive"/> class.
+        /// </summary>
+        public Primitive()
+        {
+            Base = UnityEngine.Object.Instantiate(ToysHelper.PrimitiveBaseObject);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Primitive"/> class from a <see cref="PrimitiveObjectToy"/>.
+        /// </summary>
+        /// <param name="toy">The toy to be wrapped.</param>
+        public Primitive(PrimitiveObjectToy toy)
+        {
+            Base = toy;
+            Collidable = toy.NetworkPosition.x < 0 && toy.NetworkPosition.y < 0 && toy.NetworkPosition.z < 0;
+        }
+
+        /// <summary>
+        /// Gets the base <see cref="PrimitiveObjectToy"/>.
+        /// </summary>
+        public PrimitiveObjectToy Base { get; }
+
+        /// <summary>
+        /// Gets or sets the material color of the primitive.
+        /// </summary>
+        public Color Color
+        {
+            get => Base.NetworkMaterialColor;
+            set => Base.NetworkMaterialColor = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the type of the primitive.
+        /// </summary>
+        public PrimitiveType Type
+        {
+            get => Base.NetworkPrimitiveType;
+            set => Base.NetworkPrimitiveType = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the position of the primitive.
+        /// </summary>
+        public Vector3 Position
+        {
+            get => Base.NetworkPosition;
+            set => Base.NetworkPosition = value;
+        }
+
+        /// <summary>
+        /// Gets or sets the rotation of the primitive.
+        /// </summary>
+        public Quaternion Rotation
+        {
+            get => Base.NetworkRotation.Value;
+            set => Base.NetworkRotation = new LowPrecisionQuaternion(value);
+        }
+
+        /// <summary>
+        /// Gets or sets the scale of the primitive.
+        /// </summary>
+        public Vector3 Scale
+        {
+            get => Base.NetworkScale;
+            set
+            {
+                Base.NetworkScale = value;
+                RefreshCollidable();
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the amount of movement smoothening on the light.
+        /// You can assign it as a byte (0-255), where higher values mean less smooth movement.
```

Comment:
Remove the byte thing, it was just for you to know, just write that higher values mean less smoother and 60 is the ideal